.eslintrc

```diff
@@ -1,10 +1,17 @@
 {
-  "extends": "godaddy",
-  "rules": {
-    "strict": 0,
-    "no-shadow": 0
-  },
-  "parserOptions": {
-    "ecmaVersion": 2017
-  }
-}
+    "env": {
+        "es2020": true,
+        "node": true,
+        "mocha": true
+    },
+    "extends": "godaddy",
+    "parserOptions": {
+        "ecmaVersion": 11,
+        "sourceType": "module"
+    },
+    "rules": {
+        "no-process-env": "off",
+        "no-console": "off",
+        "guard-for-in": "off"
```

Comment:
This is a good rule to preserve; I'd suggest replacing all `for..in` loops with better alternatives.